src/asset/unit.ts

```diff
@@ -0,0 +1,23 @@
+import {Asset, Wallet} from "../interfaces";
+import ApiPromise from "@polkadot/api/promise";
+import {getBalance} from "../rpc/substrate/getBalance";
+import {getAddress} from "../rpc/getAddress";
+import {executeAssetOperation} from "./executeAssetOperation";
+
+export function unitAsset(balance: string, address: string): Asset {
+  return {
+    balance: balance,
+    customViewUrl: `https://polkascan.io/pre/kusama/account/${address}`,
+    decimals: 0,
+    identifier: 'unit-test-asset',
+    image: '',
+    symbol: 'KSM',
+  };
+}
+
+export async function manageUnitAsset(wallet: Wallet, api: ApiPromise, method: "update" | "add"): Promise<Asset> {
```

Comment:
- rename this to createPolkadotAsset
- later on we will add onether method updateAssetBalance